docker-images/baleine/Dockerfile

```diff
@@ -1,5 +1,28 @@
 FROM ubuntu:latest
 
+# Install necessary binaries for tutorials
+RUN apt update -y && apt upgrade -y && apt install kmod iw iputils-ping openssh-client openssh-server bash netcat iproute2 net-tools tcpdump dnsutils netcat nano vim -y
+
+# Setup Bash prompt to not get lost when in interactive mode
+RUN echo "PS1='[CONTAINER] ${debian_chroot:+($debian_chroot)}\u@\h:\w\$ '" | tee /root/.bashrc > /dev/null
+
+#Setting up OpenSSH server
+RUN sed -ri 's/^#?PermitRootLogin\s+.*/PermitRootLogin yes/' /etc/ssh/sshd_config
+RUN sed -ri 's/UsePAM yes/#UsePAM yes/g' /etc/ssh/sshd_config
```

Comment:
it's generally desirable to make this kind of script idempotent (i.e. that running it twice or more is like running it once) 
I know this is not quite relevant here, but still, it's a good habit
you never know in what context this can be adapted to...

in this respect you should protect this replacement, make it happen only when `UsePam` is at the beginning of the line, this way the second time around it won't trigger